astronomer/providers/apache/hive/sensors/hive_partition.py

```diff
@@ -10,11 +10,16 @@
 
 class HivePartitionSensorAsync(HivePartitionSensor):
     """
-    Waits for a given partition to show up in Hive.
-    Note: HivePartitionSensorAsync uses implya library instead of pyhive
-    since pyhive is currently unsupported. Refer https://github.com/dropbox/PyHive.
-    Since we use implya library, please set the connection to use the port 10000 instead of 9083
-    The sensor currently supports auth_mechansim='PLAIN' only.
+    .. note:: HivePartitionSensorAsync uses implya library instead of pyhive
+       since pyhive is currently `unsupported <https://github.com/dropbox/PyHive>`_.
+
+       Since we use implya library, please set the connection to use the port ``10000`` instead of ``9083``
+       The sensor currently supports ``auth_mechansim='PLAIN'`` only.
+
+       The library version of hive and hadoop in ``Dockerfile`` should match the remote cluster where the
+       hadoop and hive is running.
+
+    Waits for a given partition to show up in Hive table.
```

Comment:
`Waits for a given partition to show up in Hive.` should come first in the docstring, and then you can mention the note

Reply:
Moved to the beginning of docstring.
.